docs/source/contributing.rst

```diff
@@ -1,3 +1,87 @@
 .. _contributing:
 
-.. include:: ../../../CONTRIBUTING.rst
+Contributing to Wagtailtrans
+============================
+
+Thank you for considering to help wagtailtrans.
+
+We appreciate all kinds of support, whether it’s on bug reports, code,
+documentation, tests or feature requests.
+
+Issues
+------
+
+Our Github `issue tracker <https://github.com/LUKKIEN/wagtailtrans/issues>`_ is the preferred channel for bug reports,feature requests and submitting pull-requests. Please don’t use this for support of any kind. When creating new issues please provide as much relevant context as possible. Even your detailed comments on existing issues are a big help.
```

Comment:
Please remove: `Please don’t use this for support of any kind. `. At this moment in time our Issue tracker is also the place for questions, since we are not using a different channel for this.